src/components/Comments/CommentsThreadList.graphql

```diff
@@ -0,0 +1,40 @@
+query CommentThreadsList($resourceId: ID!, $input: CommentThreadListInput) {
+  commentThreads(resource: $resourceId, input: $input) {
+    total
+    hasMore
+    items {
+      ...CommentThreadProps
+    }
+  }
+}
+
+fragment CommentProps on Comment {
```

Comment:
"Props" suffix here isn't really idiomatic. This seems to be mainly used in the `CommentItem` component - can we rename to `commentItem` and move to `CommentItem/commentItem.graphql`?